### PR TITLE
Clarify Pronunciation

### PR DESCRIPTION
**Pull request by walston**
## Summary
Include IPA pronunciation and example of a standard english language rhyme

## Related Issues
N/A

### CODE REVIEW

**Comment by jeanm** [comment]
If Xi really is pronounced like _sigh_, then I don't think ⟨ɪ̯⟩ is correct. That would indicate a semi-vowel, and "igh" in _sigh_ isn't one. I would rather transcribe it as /zaɪ/, placed between slashes (cf. [the phonetic transcription of sigh on Wiktionary](https://en.wiktionary.org/wiki/sigh#Pronunciation).

**Comment by raphlinus** [comment]
I agree with @jeanm here. The "inverted breve below" is overly pedantic, and according to [Wikipedia](https://en.wikipedia.org/wiki/Diphthong#Transcription) is really only indicated when there is a contrastive vowel sequence in the language, which there isn't here.

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <a href="https://github.com/xi-editor/xi-editor">Xi Editor</a>
 </h1>
 
-<p align="center"><em>(pronounced "Zigh")</em></p>
+<p align="center"><em>pronounced "zaɪ̯"</em>; rhymes with "sigh", but starts with "z" from "zoo"</p>
```
Comment (cmyr) [inline README.md]:
```suggestion
<p align="center"><em>pronounced</em> /zaɪ/; <em>rhymes with "sigh"</em></p>
```

Comment (walston) [reply README.md]: I dig this solution.

**Pull request closed without merging**